tests/system/memory_leak/test_memcheck.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# Tests for checking meomory leaks.
+
+source config.sh
+
+export FLEDGE_ROOT=$(pwd)/fledge
+
+FLEDGE_TEST_BRANCH=$1     #here fledge_test_branch means branch of fledge repository that is needed to be scanned, default is devops
+
+cleanup(){
+  # sudo rm -rf /usr/local/fledge
+  rm -rf /tmp/*valgrind*.log /tmp/*valgrind*.xml
+  echo "Removig Repositories..."
```

Comment:
"Removing cloned repositories and temp files"?

Reply:
Sure. I was actually pointing to the typo here "Removig", so you may as well update the comment, if you want.

Reply:
ok, I'm doing the changes